analysis/R/analysis_tool.R

```diff
@@ -16,6 +16,7 @@
 
 library(optparse)
 
+setwd("~/git/google_dev_rappor/analysis/R/")
```

Comment:
There is probably some issue relating to ../.. below, but I don't think this fix belongs here, since analysis_tool is separate.  Please revert this and we can deal with this separately.